app/views/resources/_work_version.html.erb

```diff
@@ -13,25 +13,17 @@
       ) %>
 
   <% if policy(work_version.work).edit? %>
```

Comment:
Same thing here. The `render?` method would control whether or not to show the button.